WordPress.gitignore

```diff
@@ -1,19 +1,23 @@
-*.log
-wp-config.php
-wp-content/advanced-cache.php
-wp-content/backup-db/
-wp-content/backups/
-wp-content/blogs.dir/
-wp-content/cache/
-wp-content/upgrade/
-wp-content/uploads/
-wp-content/mu-plugins/
-wp-content/wp-cache-config.php
+# ignore everything in the root except the "wp-content" directory.
+!wp-content/
+
+# ignore everything in the "wp-content" directory, except:
+# "mu-plugins", "plugins", "themes" directory
+wp-content/*
+!wp-content/mu-plugins/
+!wp-content/plugins/
+!wp-content/themes/
+
+# ignore these plugins
 wp-content/plugins/hello.php
 
-/.htaccess
-/license.txt
-/readme.html
-/sitemap.xml
-/sitemap.xml.gz
+# ignore specific themes
+wp-content/themes/twenty*/
 
+# ignore node dependency directories
+node_modules/
```

Comment:
Does Wordpress use node_modules in the root?

Reply:
Yes. 
Here you will find the list of packages using by WordPress:
https://core.trac.wordpress.org/browser/trunk/package.json